config/settings/common.py

```diff
@@ -205,6 +222,19 @@
 # ------------------------------------------------------------------------------
 ROOT_URLCONF = "config.urls"
 
+# ADMIN_AREA_PREFIX = env("DJANGO_ADMIN_URL", default="admin/", cast=url_prefix)
+
+RESTRICTED_PREFIXES = env(
+    "RESTRICTED_PREFIXES", default=["admin/", "api/"], cast=url_prefix_list
+)
+
+ADMIN_API_ALLOWED_SUBNETS = env(
+    "ADMIN_API_ALLOWED_SUBNETS",
+    default="127.0.0.1/32",
```

Comment:
For backwards compatibility this should probably default to 0.0.0.0/0

We want to be more restrictive for our own deployments, but we don't want to catch other installations of MetaCI off guard.

Reply:
Okay.